shared/domain/src/main/java/gov/va/vro/model/mas/MasAutomatedClaimPayload.java

```diff
@@ -22,7 +24,6 @@ public class MasAutomatedClaimPayload implements Auditable {
   public static final String DISABILITY_ACTION_TYPE_NEW = "NEW";
   public static final String DISABILITY_ACTION_TYPE_INCREASE = "INCREASE";
   public static final String AGENT_ORANGE_FLASH_ID = "266";
```

Comment:
Is the property, agentOrange, defined in application.yml the same as this constant? If yes, can they be defined once and shared?